repo/datastore.go

```diff
@@ -63,6 +64,13 @@ type PeerStore interface {
 	DeleteByAddress(address string) error
 }
 
+type BotStore interface {
```

Comment:
This could be moved in to the core interfaces along with this whole `Datastore` interface (eventually).

Reply:
👍